app/controllers/api/v1/book_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class BookController < ApplicationController
+      before_action :authenticate_user!
```

Comment:
move this line to ApplicationController

Reply:
If I move that line to ApplicationController I can't log in

![Captura de Pantalla 2021-06-09 a la(s) 11 01 00](https://user-images.githubusercontent.com/52846611/121369327-20b10b00-c912-11eb-916a-ae688ea9f592.png)

Reply:
you should create an ApiController to manage all API configurations and all API controllers inherit of this, and this inherit of ApplicationController